paymentsource/types.go

```diff
@@ -0,0 +1,6 @@
+package paymentsource
+
+const (
+	BitcoinReceiver = "bitcoin_receiver"
```

Comment:
yes, they should be typed!

My thinking for putting these into the paymentsource package was that the terms I wanted were already defined in the top level stripe package. I could certainly do `BitcoinReceiverType` or `CardSourceType` or something to that effect and define them in the top level `payment_source.go` file -- is that a more reasonable approach?

Thanks much for looking this over, btw.